tests/constant_time/kem/issues.json

```diff
@@ -1,6 +1,7 @@
 {
-  "BIKE-L1": ["bike_has_no_timing_protections"],
-  "BIKE-L3": ["bike_has_no_timing_protections"],
+  "BIKE-L1": [],
```

Comment:
As Michael points out, removing this means that constant-time tests are failing.